README.md

```diff
@@ -42,13 +41,13 @@ into your HTML pages:
 Add this to your `<head>` element:
 
 ```html
-<link rel="stylesheet" href="/path/to/your/assets/css/uswds-0.9.0.css">
+<link rel="stylesheet" href="/path/to/your/assets/css/uswds.css">
```

Comment:
@msecret it changed due to maintaining the documentation using the version number. It's not a good idea to have to update version numbers everywhere whenever in the documentation when we update the package's version. Also, it could be a user pain to have our users update and then break their asset-paths.